priv/tools/internal/riak_cs_inspector.erl

```diff
@@ -84,10 +84,7 @@
          {help, $H, "help", boolean, "Show this message"}
         ]).
 
-% TODO
-%-define(sort_opt_spec, [{sort, $s, "sort", undefined,
-                            %"determine to sort results (not implemented yet)"}]).
--define(sort_opt_spec, []).
+-define(sort_opt_spec,   [{sort, $s, "sort", undefined, "Determine to sort results instead of streaming list keys"}]).
```

Comment:
At 4th item, Instead of `undefined`, `{boolean, false}` may be better.
The help message can be more simple? For example "Sort the results".